src/NzbDrone.Core/Parser/Parser.cs

```diff
@@ -22,6 +23,29 @@ public static class Parser
 //                new Regex(@"^(?:(?<absoluteepisode>\d{2,3})(?:_|-|\s|\.)+)+(?<title>.+?)(?:\W|_)+(?:S?(?<season>(?<!\d+)\d{1,2}(?!\d+))(?:(?:\-|[ex]|\W[ex]){1,2}(?<episode>\d{2}(?!\d+)))+)",
 //                          RegexOptions.IgnoreCase | RegexOptions.Compiled),
 
+                // Track with artist (01 - artist - trackName)
```

Comment:
I would suggest creating a new Array with regexes to encompass these, so the code becomes more legible. Also it then does not have to test regexes, that are not gonna work anyways.